### ISSUE TITLE
Invalid win_addr for __ct__10LightSheetFv

### ISSUE THREAD

**Issue by Daniels118** [openblack/bw1-decomp#3]
The value of the field `win_addr` for function `__ct__10LightSheetFv` is not a valid address in `openblack.exe`.
The current value is 540265 (hex `00083E69`), the code actually starts at `00401000`.

**Comment by bwrsandman** [comment]
I definitely have that address. Code is between these values `am:00401000-ram:008a8fff `

![image](https://github.com/user-attachments/assets/8693d1f1-d8ff-44f8-80dc-3b15c0e28c2d)


MD5 | 17a530080d420c61c48e250974dcc776
-- | --


```c++
                             **************************************************************
                             *                          FUNCTION                          *
                             **************************************************************
                             undefined FUN_0083e690()
                               assume FS_OFFSET = 0xffdff000
             undefined         AL:1           <RETURN>
                             FUN_0083e690                                    XREF[5]:     FUN_004244d0:004244d3(c), 
                                                                                          FUN_00456640:0045684f(c), 
                                                                                          FUN_00688140:00688167(c), 
                                                                                          FUN_00726690:00726696(c), 
                                                                                          FUN_00780af0:00780af6(c)  
        0083e690 8b c1           MOV        EAX,ECX
        0083e692 33 c9           XOR        ECX,ECX
        0083e694 89 48 20        MOV        dword ptr [EAX + 0x20],ECX
        0083e697 89 48 24        MOV        dword ptr [EAX + 0x24],ECX
        0083e69a 89 48 28        MOV        dword ptr [EAX + 0x28],ECX
        0083e69d 89 48 40        MOV        dword ptr [EAX + 0x40],ECX
        0083e6a0 89 48 44        MOV        dword ptr [EAX + 0x44],ECX
        0083e6a3 89 48 48        MOV        dword ptr [EAX + 0x48],ECX
        0083e6a6 89 48 4c        MOV        dword ptr [EAX + 0x4c],ECX
        0083e6a9 c3              RET
}
```

**Comment by Daniels118** [comment]
First, we don't have the same version of the executable, the checksums don't match (I'm using the version with the fanpatch 1.42).
```
.text:00401000 ; Input SHA256 : 3F58BC02A6C421B4E66EF08716646BF0E1FA55AB292786EDB521D7596ED48ABA
.text:00401000 ; Input MD5    : 44DCF02750625F0775126DCA687C4A1D
.text:00401000 ; Input CRC32  : 15811248
.text:00401000 ; Compiler     : Visual C++
.text:00401000
.text:00401000 ; File Name   : E:\Games\Black and White\runblack.exe
.text:00401000 ; Format      : Portable executable for 80386 (PE)
.text:00401000 ; Imagebase   : 400000
.text:00401000 ; Timestamp   : 3D0ECF82 (Tue Jun 18 06:13:22 2002)
.text:00401000 ; Section 1. (virtual address 00001000)
.text:00401000 ; Virtual size                  : 004A795D (4880733.)
.text:00401000 ; Section size in file          : 004A8000 (4882432.)
.text:00401000 ; Offset to raw data for section: 00001000
.text:00401000 ; Flags 60000020: Text Executable Readable
.text:00401000 ; Alignment     : default
.text:00401000 ; OS type         :  MS Windows
.text:00401000 ; Application type:  Executable 32bit
```

But the problem isn't the file version (I guess the patch doesn't change the location of the existing code), I guess there is a problem in your DB or with your json exporter.
The current value in the json file is 540265 which in hex is `00083e69`, but the function you posted is at `0083e690`, please note there is one more zero. The shift occurrs during the export phase, before the value is converted to base 10, so most likely you have a missing zero in your DB, or the exporter is not reading the whole hex number.

**Comment by bwrsandman** [comment]
There is no value 540265 in the json right now... It's `8644240` at the moment which is `0x83E690` with the last 0.

**Comment by bwrsandman** [comment]
I don't really remember which version I have. I think it's 1.41 but it might be 1.20

**Comment by Daniels118** [comment]
My fault, I was stick to the initial version, you fixed this in d237ebf92a08e93cc45d4252ca69d6ff25f7f4d1
